tool/stolonctl/internal/pkg/cluster/status.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/gravitational/stolon-app/tool/stolonctl/internal/pkg/defaults"
+	"github.com/gravitational/stolon-app/tool/stolonctl/internal/pkg/kubernetes"
+
+	"github.com/gravitational/stolon/pkg/cluster"
+	"github.com/gravitational/stolon/pkg/store"
+	"github.com/gravitational/trace"
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+// Pods returns list of keeper and sentinel pods
+func Pods(client *kubernetes.Client, config *Config) ([]v1.Pod, error) {
+	pods, err := client.Pods(config.KeepersPodFilterKey, config.KeepersPodFilterValue, config.Namespace)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	sentinelPods, err := client.Pods(config.SentinelsPodFilterKey, config.SentinelsPodFilterValue, config.Namespace)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	pods = append(pods, sentinelPods...)
+	return pods, nil
+}
+
+// GetStatus returns status of Stolon cluster
+func GetStatus(pods []v1.Pod, config *Config) (*Status, error) {
+	var podsStatus []kubernetes.PodStatus
+	for _, pod := range pods {
+		podIP := pod.Status.PodIP
+		if podIP == "" {
+			podIP = "<none>"
+		}
+		podState, totalC, readyC := kubernetes.DeterminePodStatus(pod)
```

Comment:
I though `totalC` was a channel - 'C' suffix is usually used with channel variable names.
`containers` and `readyContainers`?